scripts/run.sh

```diff
@@ -81,12 +81,10 @@ cat -n <<EOF
   one compile  - Compile one component (including tests)                     - mvn clean test-comple -pl component...
   one test  IT - Verify Integration-Tests of one component                   - mvn verify -pl component -Dit.Test=...
   one jetty IT - Start Jetty Server on one component IT module               - mvn jetty:run -pl component ...
-  one sauce IT - Verify Integration-Tests of one component in SauceLabs      - mvn verify -pl component -Dsauce.user ...
```

Comment:
Please update the corresponding list of options below to remove `4) askSauce;` and `9) askSauce`.
And the numbers for all the renaming commands then have to be then changed accordingly.